src/TLuaInterpreter.h

```diff
@@ -614,7 +615,7 @@ public slots:
     static std::tuple<bool, int> getWatchId(lua_State*, Host&);
     bool loadLuaModule(QQueue<QString>& resultMsgQueue, const QString& requirement, const QString& failureConsequence = QString(), const QString& description = QString(), const QString& luaModuleId = QString());
     void insertNativeSeparatorsFunction(lua_State* L);
-
+    static void pushLabelPropertiesToLua(lua_State* L, TMapLabel label);
```

Comment:
MapLabel, since we have a normal label as well

Reply:
corrected.